web-animations-1/Overview.bs

```diff
@@ -6100,9 +6100,12 @@ The {{KeyframeEffect}} interface {#the-keyframeeffect-interface}
       (CompositeOperationOrAuto or sequence<CompositeOperationOrAuto>) composite = [];
     };
 
-    typedef (sequence<Keyframe> or PropertyIndexedKeyframes) KeyframeArgument;
+    typedef (sequence<Keyframe?> or PropertyIndexedKeyframes) KeyframeArgument;
   </xmp>
 
+  Null and undefined items in an array of keyframes
+  are converted to a keyframe with no animation property.
```

Comment:
I wonder if this note is necessary? (If it is, I think we mostly try to mark up `null` and `undefined` as `<code class=esvalue>null</code>` etc.)

Reply:
I removed it.

`null` and `undefined` are always unintentional, I think. Conscientious authors would first check [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats), which does not define the specified behavior, and then the spec. They would read the non-normative introduction and find the (corrected) conceptual type. So at this point they know that these values are not errors. They might *suspect* the actual behavior because it seems to be the *least surprising* one. But then they would check the procedure, which is not easy to understand, but that should be fine...